libs/util/http.client.js

```diff
@@ -26,6 +26,17 @@ var DEFAULT_CONFIG = {
 
 var INITIAL_ATTEMPT = 0;
 
+function parseResponse(response) {
```

Comment:
this was in fetcher.client.js but not needed there?

Reply:
It is not that it's not needed. But it's inconsistent that we parse error responses inside the http module but normal responses we parse inside the fetcher module. Now it's everything in one place (and we can potentially use `fetch().(response => response.json()))` instead of `text()` (see: https://github.com/yahoo/fetchr/blob/80923dc7d908aae547e0a8b60528fecfc0943177/libs/util/http.client.js#L223). This change was not really related to this PR but it was lingering in my machine for an while.